testing/py-anyjson/APKBUILD

```diff
@@ -1,22 +1,29 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-anyjson
 _pkgname=anyjson
 pkgver=0.3.3
-pkgrel=1
+pkgrel=2
 pkgdesc="Wraps the best available JSON implementation available in a common interface"
 url="https://bitbucket.org/runeh/anyjson/"
 arch="noarch"
 license="BSD"
+checkdepends="py-future py-nose"
 makedepends="python2-dev python3-dev py-setuptools"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
+	python3.patch"
 builddir="$srcdir"/$_pkgname-$pkgver
 
 build() {
-	cd "$builddir"
```

Comment:
why you removed that? the same should be added to `check()` - all this commands should start in bulddir

Reply:
I thought newest abuild will change to $builddir automatically, but `cd $builddir` should not be removed before 3.9-stable. Sorry for that. Readded.